sql-jdbc/build.gradle

```diff
@@ -24,7 +24,7 @@ plugins {
 group 'org.opensearch.client'
 
 // keep version in sync with version in Driver source
-version '2.2.0.0'
+version '2.3.0.0'
```

Comment:
ok but don't need to bump drivers, latest released jdbc version is 1.1.0.1 so next release will likely be 1.2.0.0

Reply:
change to 1.1.0.1